source/modules.tex

```diff
@@ -153,7 +153,7 @@
 
 \pnum
 A \grammarterm{module-declaration}
-that contains neither \tcode{export}
+that contains neither an \grammarterm{export-keyword}
```

Comment:
This one probably needs to stay as-is.

Reply:
By "this one" do you mean "contains" or `export`?

Reply:
I mean "contains".